dbt/models/sec10k/core_sec10k__assn_exhibit_21_subsidiaries_and_eia_utilities/schema.yml

```diff
@@ -5,10 +5,13 @@ sources:
       - name: core_sec10k__assn_exhibit_21_subsidiaries_and_eia_utilities
         data_tests:
           - expect_columns_not_all_null
-          - check_row_counts_per_partition:
+          - dbt_expectations.expect_table_row_count_to_be_between:
+              description: Nightly builds and local ETL yield slightly different output
+              config:
+                enabled: "{{ target.name == 'etl-full'}}"
               arguments:
-                table_name: core_sec10k__assn_exhibit_21_subsidiaries_and_eia_utilities
-                partition_expr:
+                min_value: 5543
+                max_value: 5564
```

Comment:
For a month or so the row counts on this table had stabilized, but now they're different local and remote again, so back to the range.